hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/validators/ApproveAllowanceValidator.java

```diff
@@ -129,7 +130,7 @@ private void validateFungibleTokenAllowances(
         for (final var allowance : tokenAllowances) {
             final var owner = allowance.owner();
             final var spender = allowance.spenderOrThrow();
-            final var token = tokenStore.get(allowance.tokenIdOrThrow());
+            final var token = tokenStore.get(allowance.tokenIdOrElse(TokenID.DEFAULT));
```

Comment:
Shouldn't this fail in pureChecks ?

Reply:
Agreed that is more consistent @Neeharika-Sompalli , fixed!